lms/djangoapps/badges/tests/test_models.py

```diff
@@ -32,7 +32,7 @@ def get_image(name):
     """
     Get one of the test images from the test data directory.
     """
-    return ImageFile(open(TEST_DATA_ROOT / 'badges' / name + '.png', mode='rb'))  # lint-amnesty, pylint: disable=bad-option-value, open-builtin
+    return ImageFile(open(f'{TEST_DATA_DIR}/badges/{name}.png', mode='rb'))  # lint-amnesty, pylint: disable=bad-option-value, open-builtin
```

Comment:
Django doesn't allow to store the absolute path in `ImageField`

Source: https://github.com/django/django/blob/2.2.23/django/core/files/utils.py#L17